src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1525,6 +1527,25 @@ private function dumpValue($value, $interpolate = true)
             }
 
             return sprintf('array(%s)', implode(', ', $code));
+        } elseif ($value instanceof LocatorArgument) {
+            $services = $value->getValues();
+            $servicesMap = array();
+            $getterCode = 'function ($id) {'."\n";
+            foreach ($services as $k => $ref) {
+                $servicesMap[] = sprintf("'%s' => '%s'", $k, $ref);
```

Comment:
instead of having to iterate through the services, for the construction of the service map, you should use a `var_export` : `var_export($services);`